apprentice-react-o-shop/src/utils/history.js

```diff
@@ -0,0 +1,5 @@
+import { createBrowserHistory } from 'history';
+
+const history = createBrowserHistory()
+
+export default history;
```

Comment:
Please add an extra line at the end of the file. Anytime you see this red icon it's github complaining about that

Reply:
Really should make myself a linter setting for this 😆

Reply:
Turns out there is a VSCode setting for it that is now turned on!